frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.module.css

```
@@ -23,6 +23,10 @@
   border-bottom: 1px solid var(--node-border);
 }
 
+.columnWrapper:last-child {
+  border-radius: var(--border-radius-md);
```

As the upward radius is not needed, the following may be acceptable:

```suggestion
  border-bottom-left-radius: var(--border-radius-md);
  border-bottom-right-radius: var(--border-radius-md);
```

https://developer.mozilla.org/en-US/docs/Web/CSS/border-bottom-left-radius

---

Oh, exactly...🤦‍♀️.Thank you!
Fixed: https://github.com/liam-hq/liam/pull/206/commits/81a7c5e2e4522f004dca426ff57e07545bd23c06